### PR TITLE
Fix fuse docs and script

### PR DESCRIPTION
**Pull request by LuQQiu**
Remove the solid requirement for setting of JAVA_HOME in alluxio-fuse script

### CODE REVIEW

**Comment by LuQQiu** [comment]
@madanadit PTAL, thanks!

File: docs/en/api/POSIX-API.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ For example, the following command will mount the Alluxio path `/people` to the
 in the local file system.
 
 ```bash
+bin/alluxio fs mkdir /people
```
[the remark this answers is not in the document]
Comment (LuQQiu) [reply docs/en/api/POSIX-API.md]: The `alluxio-fuse mount` will not create the folder in alluxio.

[the remark this answers is not in the document]
Comment (madanadit) [reply docs/en/api/POSIX-API.md]: @LuQQiu i'm talking about the following `sudo mkdir -p /mnt/people`

[the remark this answers is not in the document]
Comment (LuQQiu) [reply docs/en/api/POSIX-API.md]: POSIX API require the mount point to be an existing directory, we don't create one for users when mounting

**Comment by madanadit** [comment]
alluxio-bot, merge this please.

**Comment by madanadit** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please.

**Pull request merged**